src/sysdolphin/baselib/class.c

```diff
@@ -1,25 +1,40 @@
 #include <sysdolphin/baselib/class.h>
 
 #include <dolphin/os/os.h>
+#include <string.h>
+#include <sysdolphin/baselib/hash.h>
 #include <sysdolphin/baselib/memory.h>
 #include <sysdolphin/baselib/object.h>
 
 void _hsdClassInfoInit(void);
-HSD_ClassInfo hsdClass = { _hsdClassInfoInit };
+HSD_ClassInfo hsdClass = {
+    { _hsdClassInfoInit, FLAGS_NONE, NULL, NULL, 0, 0, NULL, NULL, NULL, 0, 0 },
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+};
```

Comment:
```suggestion
HSD_ClassInfo hsdClass = { _hsdClassInfoInit };
```
Rest of class is guaranteed to be zero-initialized